schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a context related to a screen viewed.",
+    "self": {
+        "vendor": "com.snowplowanalytics.mobile",
+        "name": "screen",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string",
+            "maxLength": 100,
+            "description": "The name of the screen viewed."
+        },
+        "type": {
+            "type": "string",
+            "maxLength": 100,
+            "description": "The type of screen that was viewed e.g feed / carousel."
+        },
+        "screen_session_id": {
+            "type": "string",
+            "maxLength": 100,
+            "description": "An ID that can be used in lieu of the name property, usually regenerated upon each session."
```

Comment:
Why would the screen ID be regenerated each session? I would assume there is a 1:1 mapping of screen IDs to screen name and screen name would persist past the session?